src/Integration.UnitTests/LocalServices/ProjectSystemHelperTests.cs

```diff
@@ -730,5 +736,177 @@ int IVsShell.UnadviseShellPropertyChanges(uint dwCookie)
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <MyGroup Include=""MyValue1"" />
+    <MyGroup Include=""MyValue2"" />
+  </ItemGroup>
+</Project>";
+
+            var project = new ProjectMock("test.csproj");
+            fileSystem.Setup(x => x.File.ReadAllText(project.FilePath)).Returns(projectXml);
+
+            var actual = testSubject.DoesExistInItemGroup(project, "MyGroup", "MyValue");
+            actual.Should().BeFalse();
+        }
+
+        [TestMethod]
+        public void DoesExistInItemGroup_ProjectHasOneCorrectIncludeThatIsCommentedOut_False()
+        {
+            var projectXml = @"<Project Sdk=""Microsoft.NET.Sdk"">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <!--<MyGroup Include=""MyValue"" />-->
+</ItemGroup>
+</Project>";
+
+            var project = new ProjectMock("test.csproj");
+            fileSystem.Setup(x => x.File.ReadAllText(project.FilePath)).Returns(projectXml);
+
+            var actual = testSubject.DoesExistInItemGroup(project, "MyGroup", "MyValue");
+            actual.Should().BeFalse();
+        }
+
+        [TestMethod]
+        public void DoesExistInItemGroup_ProjectHasOneCorrectInclude_CorrectValue_True()
+        {
+            var projectXml = @"<Project Sdk=""Microsoft.NET.Sdk"">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <MyGroup Include=""MyValue"" />
+  </ItemGroup>
+</Project>";
+
+            var project = new ProjectMock("test.csproj");
+            fileSystem.Setup(x => x.File.ReadAllText(project.FilePath)).Returns(projectXml);
+
+            var actual = testSubject.DoesExistInItemGroup(project, "MyGroup", "MyValue");
+            actual.Should().BeTrue();
+        }
+
+        [TestMethod]
+        public void DoesExistInItemGroup_ProjectHasMultipleCorrectIncludes_HasCorrectValue_True()
+        {
+            var projectXml = @"<Project Sdk=""Microsoft.NET.Sdk"">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <MyGroup Include=""MyValue"" />
+    <MyGroup Include=""MyValue2"" />
+  </ItemGroup>
+</Project>";
+
+            var project = new ProjectMock("test.csproj");
+            fileSystem.Setup(x => x.File.ReadAllText(project.FilePath)).Returns(projectXml);
+
+            var actual = testSubject.DoesExistInItemGroup(project, "MyGroup", "MyValue");
+            actual.Should().BeTrue();
+        }
+
+        [TestMethod]
+        public void DoesExistInItemGroup_ProjectHasMultipleCorrectIncludesInDifferentItemGroups_HasCorrectValue_True()
+        {
+            var projectXml = @"<Project Sdk=""Microsoft.NET.Sdk"">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <MyGroup Include=""MyValue1"" />
+  </ItemGroup>
+  <ItemGroup>
+    <MyGroup Include=""MyValue"" />
```

Comment:
XML Attributes can be single-quoted or double-quoted, so a test with single-quotes would be good.